src/components/Users/AddUser.js

```diff
@@ -1,20 +1,51 @@
-import React from "react";
+import React, { useState } from "react";
 
 import Card from "../UI/Card";
 import Button from "../UI/Button";
 import styles from "./AddUser.module.css";
 
 const AddUser = (props) => {
+  const [enteredUserName, setEnteredUserName] = useState("");
+  const [enteredUserAge, setEnteredUserAge] = useState("");
+
   const onSubmitHandler = (event) => {
     event.preventDefault();
+    if (
+      enteredUserName.trim().length === 0 ||
+      enteredUserAge.trim().length === 0
+    ) {
+      return;
+    }
```

Comment:
Always make gap(hit enter) after if condition. For ex:
if (a) {
console.log('hello');
}

if (b) {
//do something here
}

Reply:
https://javascript.plainenglish.io/15-rules-for-writing-clean-javascript-8e2b2b426515